### PR TITLE
Grid of cards

### PR DESCRIPTION
**Pull request by matthewcrist**
This adds a card and grid of cards component to the pattern library. It also tweaks the 3 column grid.

### CODE REVIEW

File: stylesheets/components/card/_card.styl [hunk 1]
```diff
@@ -0,0 +1,50 @@
+/*
+
+   Collapsible
```
Comment (finneganh) [inline stylesheets/components/card/_card.styl]: Card

File: stylesheets/components/card/_card.styl [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+   Base:
+    cd
+
+   Elements:
```
Comment (finneganh) [inline stylesheets/components/card/_card.styl]: update these?

File: stylesheets/components/card/base.styl [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/*
+
+   SECTION HEADER
```
Comment (finneganh) [inline stylesheets/components/card/base.styl]: CARD

File: stylesheets/grid/modern/_grid.styl [hunk 4]
```diff
@@ -98,6 +98,9 @@
       margin-top: $sizing-400
 
   @media $media-xlarge
+    &--3
```
Comment (finneganh) [inline stylesheets/grid/modern/_grid.styl]: If it’s 3 columns wide, shouldn't that be 3/12 = 1/4?

Comment (matthewcrist) [inline stylesheets/grid/modern/_grid.styl]: I've been using it as g--3 = 3 items per row. g--4 = 4 items per row. etc. It's not completely intuitive, but matches what was setup originally for Boston.gov.

Comment (finneganh) [inline stylesheets/grid/modern/_grid.styl]: That's not consistent in this file. See `--6`, `--7`, `--8`, `--9`, `--10`, `--11`, and `--12` above.

**Pull request merged**